### ISSUE TITLE
Running the code on an actual device using Windows with Parallels

### ISSUE THREAD

**Issue by marcojak** [OYIon/XamarinFormsLive#47]
I have a mac with Windows running on Parallels.

If I try to use an emulator everything is working fine but if I try to use an actual device, I get this error:

[0:] XL: =============== LiveXAML Important Information ===============
[0:] XL: Failed to find find a handshake server. Switching to UDP broadcast.
Make sure you have opened 53031 and 53032 ports for incoming TCP connections on your host PC.
[0:] XL: ==============================================================

I've already opened the ports on my Windows and the firewall on my Mac is disabled. Still I get this error and no updates are shown on the device.

Is there something I can do?

**Comment by andrekiba** [comment]
Same problem here and this issue was opened more or less one year ago...could you please check?? :-(

**Comment by ionoy** [comment]
Sorry for not responding earlier, guys.

This looks like more of an issue with the network configuration than LiveXAML. For LiveXAML to work, you need the application to be executed in the same subnetwork as the development PC. If you are using the VM, make sure that networking is set up in the way that the development PC is in a common subnet.

**Comment by andrekiba** [comment]
And how to achieve this in case of real device (iPhone for example)? In Parallels if you choose "Shared Network" the virtual sub network is differente than the real one. For example you have 10.0.1.num fir the real and 10.211.55.num in Parallels... :-(
Bridged network seems the solution
https://kb.parallels.com/en/4948
but with bridged the connection between visual studio and mac agent does not work anymore.
I try to do some other tests.

**Comment by ionoy** [comment]
Yep, seems like Bridged configuration is the way to go. Not sure how to make it work with Mac Agent though.

**Comment by johndbell** [comment]
I'm having the same issue with real devices at the moment. Did anyone come up with a solution?

**Comment by stntz** [comment]
@johndbell  In my case with Parallels, I have two network interface configured for Windows, Network with Shared for iOS Build Agent to always stay connected regardless of where I am working. 
Bridged Network on WiFi Adapter for LiveXAML and 'real' device to be on the same network. make sure they are on the same WIFI. Hope that helps?
